roles/satellite-clone/tasks/main.yml

```diff
@@ -243,20 +240,20 @@
 
   - name: update katello assets
     file: src=/opt/rh/ruby193/root/usr/share/gems/gems/katello-2.2.0.93/public/assets/katello dest=/usr/share/foreman/public/assets/katello
-    when: satellite_version == 6.1
+    when: satellite_version == "6.1"
```

Comment:
what is the reason behind adding double quotes here?

Reply:
So the check_metadata_for_satellite_version module returns a string and I can't get it to return an integer. I'm guessing this has to do with how `register` works in ansible. I changed satellite_version to always be a string to match the new module and updated usage appropiately